publish-nokendra.sh

```diff
@@ -72,24 +72,14 @@ npm install
 npm run build
 popd
 
-
-echo "Downloading witch file and upload into artifacts bucket"
-curl https://saes-prod-us-east-1.s3.us-east-1.amazonaws.com/witch-0eabcaf.zip -o /tmp/witch-0eabcaf.zip
-WITCHKEY=${PREFIX_AND_VERSION}/witch-0eabcaf.zip
-aws s3 cp /tmp/witch-0eabcaf.zip s3://${BUCKET}/${WITCHKEY} || exit 1
+pushd pca-ui/src/witch
+npm install
+npm run build
+popd
 
 echo "Packaging Cfn artifacts"
 aws cloudformation package --template-file pca-main-nokendra.template --output-template-file /tmp/packaged.template --s3-bucket ${BUCKET} --s3-prefix ${PREFIX_AND_VERSION} --region ${region}|| exit 1
 
-echo "Inline edit tmp/packaged.template to replace "
-echo "   <ARTIFACT_BUCKET_TOKEN> with bucket name: $BUCKET"
-echo "   <WITCHKEY_TOKEN> with prefix: $WITCHKEY"
-cat /tmp/packaged.template | 
-sed -e "s%<ARTIFACT_BUCKET_TOKEN>%$BUCKET%g" | 
-sed -e "s%<WITCHKEY_TOKEN>%$WITCHKEY%g" > build/packaged.template
-
-aws s3 cp build/packaged.template s3://${BUCKET}/${PREFIX}/pca-main.yaml || exit 1
```

Comment:
This removed line uploads the main template to the artifacts S3 bucket.. Removing it causes the script to fail below when calling ```aws cloudformation validate-template``` since the template doesn't exist in the target S3 location.

Reply:
Done. I've added it back in